### DISCUSSION TITLE
Thread 1.0 and OpenThread interoperability

### DISCUSSION THREAD

**Discussion by ksathara** [openthread/openthread#8825] [Q&A] [answered]
Hi, We have existing devices which are based on Thread 1.0. Now the new products are based on OpenThread. Can these communicate with each other? How can we set it up?

Cheers,
Kaushalya

**Comment by jwhui** [comment, marked as answer]
Thread 1.1 was the first version that Thread Group support Thread Certification. All newer versions of Thread are tested to be backward compatible with Thread 1.1.
